packages/webpack-scaffold/src/index.ts

```diff
@@ -58,7 +58,7 @@ export function List(
     message: string,
     choices: string[],
     defaultChoice?: string,
-    skip = false,
+    skip: boolean = false, // eslint-disable-line @typescript-eslint/no-inferrable-types
```

Comment:
We have rule for it to avoid it, why you want to change it? It doesn't have sense because it is already boolean

Reply:
It was put forward by as a suggestion in the previous PR. But yeah as you said it won't be a good idea.